parm/use_cases/model_applications/tc_and_extra_tc/TCGen_fcstGFS_obsBDECK_2021season.conf

```diff
@@ -0,0 +1,162 @@
+[config]
+
+# Looping by times: steps through each 'task' in the PROCESS_LIST for each
+# defined time, and repeats until all times have been evaluated.
+LOOP_ORDER = times
+
+# 'Tasks' to be run
+PROCESS_LIST = TCGen
+
+LOOP_BY = INIT
+
+# The init time
+INIT_TIME_FMT = %Y
+INIT_BEG = 2021
+
+
+LOG_TC_GEN_VERBOSITY = 2
+
+# optional list of strings to loop over and call the tool multiple times
+# value of each item can be referenced in filename templates with {custom?fmt=%s}
+TC_GEN_CUSTOM_LOOP_LIST =
+
+
+# I/O Configurations
+
+# Location of input data directory for track data
+TC_GEN_TRACK_INPUT_DIR = {INPUT_BASE}/model_applications/tc_and_extra_tc/TCGen_fcstGFS_obsBDECK_2021season/abdeck/
+TC_GEN_TRACK_INPUT_TEMPLATE = *.dat
+
+# Location of input data directory for genesis data
+TC_GEN_GENESIS_INPUT_DIR = {INPUT_BASE}/model_applications/tc_and_extra_tc/TCGen_fcstGFS_obsBDECK_2021season/genesis/
+TC_GEN_GENESIS_INPUT_TEMPLATE = genesis*.atcf_gen
+
+TC_GEN_EDECK_INPUT_DIR = {INPUT_BASE}/model_applications/tc_and_extra_tc/TCGen_fcstGFS_obsBDECK_2021season/edeck/
+TC_GEN_EDECK_INPUT_TEMPLATE = edeck*.dat
+
+TC_GEN_SHAPE_INPUT_DIR = {INPUT_BASE}/model_applications/tc_and_extra_tc/TCGen_fcstGFS_obsBDECK_2021season/shape
```

Comment:
This directory is not listed correctly; METplus does not read the initial `*` in the TC_GEN_SHAPE_INPUT_TEMPLATE variable and instead ignores everything between the first two (so it ignores */{init?fmt=%Y}*). 

What it's searching through is actually `{INPUT_BASE}/model/applications/tc_and_extra_tc/TCGen_fcstGFS_obsBDECK_2021season/shape/gtwo*.shp`. This can likely be corrected by moving everything but the name of the file into the TC_GEN_SHAPE_INPUT_DIR.

Reply:
This comment was made in error: was testing against an older version of this file, while the current one runs fine with this setting.